genie-agent/build.gradle

```diff
@@ -23,6 +23,14 @@ java {
     }
 }
 
+dependencyManagement {
```

Comment:
you should be able to define it in the root `build.gradle` to reduce duplication (with configure subprojects each)